package.json

```diff
@@ -57,12 +57,12 @@
     },
     "devDependencies": {
         "@types/node": "^8.10.25",
-        "tslint": "^5.11.0",
+        "tslint": "^5.20.1",
         "typescript": "^2.6.1",
-        "vscode": "^1.1.22"
+        "vscode": "^0.9.9"
     },
```

Comment:
Why do we need to change these dependencies?

Reply:
I don't think so. Do you want me to downgrade it? There were some updates due to `npm audit` but i'm not sure this was one of them.

Reply:
If this particular change isn't required I would propose to revert it to the original